app/components/pages/SubmissionWizard/steps/Editors/EditorsContent.js

```diff
@@ -79,6 +86,45 @@ class EditorsContent extends React.Component {
     )
   }
 
+  handleSuggestedReviewersChanged = event => {
+    this.props.handleChange(event)
```

Comment:
Unless I'm missing something, I don't think there's a `handleChange` being passed to this component as a prop?

Reply:
It's passed down from Formik.